absl/synchronization/internal/waiter.h

```diff
@@ -40,7 +40,7 @@
 
 #if defined(ABSL_FORCE_WAITER_MODE)
 #define ABSL_WAITER_MODE ABSL_FORCE_WAITER_MODE
-#elif defined(_WIN32)
+#elif defined(_WIN32) && _WIN32_WINNT >= 0x0600
```

Comment:
How would you feel about comparing against _WIN32_WINNT_VISTA instead of the magic constant?  Looks like you'd have to conditionally #include SdkDdkVer.h.

If that's not possible or desirable, please add a comment indicating that 0x0600 corresponds to Vista.